tests/index/CMakeLists.txt

```diff
@@ -164,6 +169,7 @@ add_test (NAME test_test_index_fm_rank_dictionary COMMAND $<TARGET_FILE:test_ind
 add_test (NAME test_test_index_fm_sparse_string COMMAND $<TARGET_FILE:test_index_fm_sparse_string>)
 add_test (NAME test_test_index_base COMMAND $<TARGET_FILE:test_index_base>)
 add_test (NAME test_test_index_fm COMMAND $<TARGET_FILE:test_index_fm>)
+add_test (NAME test_test_index_bifm COMMAND $<TARGET_FILE:test_index_bifm>)
```

Comment:
Can't you add the bi-fmi to other index tests like test_test_index_base and test_test_index_fm?